workers/social/lib/social/models/invitation.coffee

```diff
@@ -113,6 +113,7 @@ module.exports = class JInvitation extends jraphical.Module
 
         { _id } = account
         { roles } = permissionAndRoles
+        groupName = client.context.group or 'koding'
```

Comment:
let's not make something like this if there is no `groupName` let's skip notifications instead.